HdProduction.HelpDesk.Infrastructure/Repositories/RepositoryBase.cs

```diff
@@ -0,0 +1,40 @@
+using System.Threading.Tasks;
+using HdProduction.HelpDesk.Domain.Contract;
+using HdProduction.HelpDesk.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace HdProduction.HelpDesk.Infrastructure.Repositories
+{
+    public abstract class RepositoryBase<T, TKey> : IRepository<T, TKey> 
+        where T : EntityBase<TKey> where TKey : struct
+    {
+        protected readonly ApplicationContext Context;
```

Comment:
Context instead of _context? Because protected instead of private?

Reply:
yep, name conventions